team/michael-town.yaml

```diff
@@ -0,0 +1,22 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Michael (Mike) Town (he/him)
+  avatar: https://avatars.githubusercontent.com/u/80553650?v=4
+  role: Research Scientist
+  organizations:
+    - name: Earth and Space Research
+  bio:
+ I am an innovative research scientist, educator, and instructional coach with professional experience in machine learning and data analysis, computational modeling, remote field work, and secondary and university-level curriculum development. Equity and social justice are at the center of both my research and education efforts.
```

Comment:
@michaeltown-phd It looks like the indentation is off here (and is causing the build to fail).

Reply:
OK, I think I fixed it. I ran a 'preview' and that seemed to succeed. Sorry!